openassessment/workflow/api.py

```diff
@@ -182,8 +182,10 @@ def get_workflow_for_submission(submission_uuid, assessment_requirements):
     return update_from_assessments(submission_uuid, assessment_requirements)
 
 
-def update_from_assessments(submission_uuid, assessment_requirements):
-    """Update our workflow status based on the status of peer and self assessments.
+def update_from_assessments(submission_uuid, assessment_requirements, force_update_score=False):
```

Comment:
I'm wondering whether this ``force_update_score`` is really necessary. Couldn't it be derived to be true if a staff score is found? It seems less brittle to me to keep that logic within this function, than to remember to pass the correct value in every place that handles staff scoring. I could also imagine that for some reason the staff score didn't get applied correctly, so a subsequent update should notice it and pick it up.